truncate/table_schema.go

```diff
@@ -163,7 +164,17 @@ func targetFilterTableSchemas(tables []*tableSchema, targetTables []string) []*t
 		isTarget[t] = true
 	}
 
-	// TODO: Add child tables that may be deleted in cascade (#18)
+	// Additionally include descendants tables that may be deleted in cascade
+	lineages := constructTableLineages(tables)
+	for _, l := range lineages {
+		if isTarget[l.tableSchema.tableName] {
+			for _, d := range l.descendants {
+				if d.isCascadeDeletable() {
+					isTarget[d.tableName] = true
+				}
+			}
+		}
+	}
```

Comment:
This is the main change in this PR. 
I am concerned that the nesting may be deep and less readable. Please suggest me if you have any good idea.

Reply:
It's heavily nested, but there is no `else` block, so I don't think it's unreadable.